src/kv/LevelDBStore.h

```diff
@@ -98,6 +98,11 @@ class LevelDBStore : public KeyValueDB {
   /// compact the underlying leveldb store
   void compact() override;
 
+  /// compact full the underlying leveldb store
```

Comment:
this comment does not look right. please either remove it or fix the syntax error.